README.md

```diff
@@ -1,7 +1,7 @@
 # jwt-go
 
 [![build](https://github.com/golang-jwt/jwt/actions/workflows/build.yml/badge.svg)](https://github.com/golang-jwt/jwt/actions/workflows/build.yml)
-[![Go Reference](https://pkg.go.dev/badge/github.com/golang-jwt/jwt.svg)](https://pkg.go.dev/github.com/golang-jwt/jwt)
+[![Go Reference](https://pkg.go.dev/badge/github.com/golang-jwt/wt/v4.svg)](https://pkg.go.dev/github.com/golang-jwt/wt/v4)
```

Comment:
Typo?

Reply:
Yes, me trying to use the GitHub UI in a hurry. Will fix it in a sec.